Rust/rust/samples/HelloWorld/hello_world_high_level_app/.vscode/launch.json

```diff
@@ -0,0 +1,20 @@
+{
+    // Use IntelliSense to learn about possible attributes.
+    // Hover to view descriptions of existing attributes.
+    // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
+    "version": "0.2.0",
+    "configurations": [
+        {
+            "name": "Launch Azure Sphere App",
+            "type": "azurespheredbg",
+            "request": "launch",
+            "args": [],
```

Comment:
You should add "program" here.  Can you use relative paths?

Reply:
This path seems to be unnecessarily long. Can we make it shorter?

Reply:
Done. Thanks for the feedback.